Orm/Xtensive.Orm/Orm/Validation/PropertyValidator.cs

```diff
@@ -39,7 +39,7 @@ public bool IsImmediate
     public bool ValidateOnlyIfModified
     {
       get => validateOnlyIfModified;
-      set {
+      init {
         if (Domain != null) {
           throw Exceptions.AlreadyInitialized(null);
```

Comment:
Error like `AlreadyInitialized` in init setter is meaningless
I'd remove it

Reply:
I agree.

Reply:
Done